xknx/devices/climate.py

```diff
@@ -83,7 +83,8 @@ def __init__(self,
                  min_temp=None,
                  max_temp=None,
                  mode=None,
-                 device_updated_cb=None):
+                 device_updated_cb=None,
+                 on_off_inverted=False):
```

Comment:
Please move it to line 83 just after `group_address_on_off_state=None,` to have all on_off things together.